core/src/main/java/org/codice/imaging/nitf/core/tre/impl/TreParser.java

```diff
@@ -79,7 +84,24 @@ public TreParser() throws NitfFormatException {
     }
 
     private void unmarshal(final InputStream inputStream) throws JAXBException {
-        tresStructure = (Tres) getUnmarshaller().unmarshal(inputStream);
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+        Document document = null;
+        try {
+            dbf.setFeature("http://xml.org/sax/features/external-general-entities", false);
+            dbf.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+            dbf.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
```

Comment:
Will an attempt be made by DocumentBuilderFactory to try to access these URLs? If it does, then this could be an issue in environments that don't have access to the Internet.